concepts/prompt.mdx

```diff
@@ -2,15 +2,79 @@
 title: "Prompt"
 ---
 
-You can draft and iterate on prompts on the Literal Prompt Playground. Once you are happy with your prompt, you can save it and use it in your code.
+You can view your prompts in the Prompt Management page. This overview shows the prompts that you have created, manage different versions of these prompts and lets you create and test prompt templates in collaboration with your teammates. In the [Literal Prompt Playground](#prompt-playground), which is connected to prompts and threads, you can debug prompts in-context, draft and iterates on prompts. Once you are happy with your prompt, you can save it and use it in your code.
 
-Moving prompts out of your code has several benefits:
+Prompts are managed in the Literal platform, which has several benefits:
 - Non technical people can draft and iterate on prompts.
 - You can deploy a new version of your prompt without deploying your code.
 - You can track which prompt version for a specific generation.
 - You can compare prompt versions to see which one performs better.
+- You can compare and test multiple LLM providers.
+- You can collaborate with the team.
+- You can do online evaluation [coming soon].
 
-## Get a Prompt
+
+## Versioning
+On the `Prompts` dashboard, you can see which prompts are present and running (`Champion Version`). By clicking on a prompt, you get to a page where you can see which versions of that prompt are saved. By clicking on a version number, you enter the [`Playground`](#prompt-playground), where you can change this prompt, or promote this prompt template to the `Champion Version`.
+
+
+## Prompt Playground
+
+The playground is an environment in which you can create, test, and debug prompts. You can access the playground via
+1. **The main menu**. This allows you to create, test and save prompts from scratch.
+2. **Prompts**. This allows you to view, edit and test your prompt template and LLM settings and manage prompt versioning.
+3. **LLM Generations**. Via Generations you can access the playground with an individual generation from a real chat conversation. This allows you to debug a single generation of the LLM in context.
+
+If you start from scratch in the Prompt Playground, you can open example templates, and iterate from there.
+
+<Frame caption="Playground Prompt Template">
+  <img src="/images/playground-prompt-template.png" alt="An example of a prompt template in the playground." />
+</Frame>
+
+### 1. Design prompt templates
+
```

Comment:
Add a small explanation on the Mustache prompt templating format with the following link: https://scalate.github.io/scalate/documentation/mustache.html#Syntax